README.md

```diff
@@ -1,7 +1,7 @@
 # Math-Magicians
```

Comment:
- [ ] According to the instructions, you should have deleted the CRA boilerplate files. Kindly delete these files as highlighted in the image below

  ![rash-cra](https://user-images.githubusercontent.com/36542986/148073326-73ac44e4-2051-4bbd-8b10-9b3c1faeaf44.jpg)